File: coralibre-sdk/sdk/src/main/java/org/coralibre/android/sdk/internal/crypto/ppcp/CryptoException.java
package org.coralibre.android.sdk.internal.crypto.ppcp;

public class CryptoException extends RuntimeException {
    CryptoException(Throwable t) {
        super(t);
    }

    CryptoException(String message) {
        super(message);
    }

    CryptoException(String message, Throwable t) {
        super(message, t);
    }
}

File: sdk/src/main/java/org/coralibre/android/sdk/internal/crypto/CryptoException.kt
package org.coralibre.android.sdk.internal.crypto

class CryptoException : RuntimeException {
    internal constructor(t: Throwable?) : super(t)
    internal constructor(message: String?) : super(message)
    internal constructor(message: String?, t: Throwable?) : super(message, t)
}
